api/policy.go

```diff
@@ -0,0 +1,49 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import "encoding/json"
+
+// PolicyService is a service that interacts with the Custom Policies
+// endpoints from the Lacework Server
+type PolicyService struct {
+	client *Client
+}
+
+type PolicyCreateResponse struct {
+	Data    []Policy `json:"data"`
+	Ok      bool     `json:"ok"`
+	Message string   `json:"message"`
+}
+
+type Policy struct {
+	PolicyID string `json:"policy_id"`
+}
+
+func (svc *PolicyService) CreatePolicy(policy string) (
```

Comment:
this looks redundant to me, if you create a new service called Policy and then a function called CreatePolicy you would call it like `client.Policy. CreatePolicy()` shouldn't be better to call this function just `Create()`?

I am assuming that Policy is in fact a separate thing from LQL, if not, why not put it inside the LQL service? That way you could do something like `client.LQL.CreatePolicy()` which is explicit and not redundant.

What do you think?

Reply:
@afiune .  You make a great point here.  I think having distinct services provides the most flexibility.  In turn we can and should simplify the function names.  I think this pattern should also be retrofitted to the `LQLService` such that we have things like `Run` vs. `RunQuery`

Reply:
Fixed.